setup.py

```diff
@@ -225,6 +225,7 @@ def write_version(filename=os.path.join(*['airflow',
 samba = ['pysmbclient>=0.1.3']
 segment = ['analytics-python>=1.2.9']
 sendgrid = ['sendgrid>=5.2.0']
+ses = ['boto3']
```

Comment:
emr has `emr = ['boto3>=1.0.0, <1.8.0']` 

so please update with version range for ses as well

Reply:
s3 is even more restrictive: `['boto3>=1.7.0, <1.8.0']`, should I set both emr and ses to the same range as s3?

Reply:
Let's remove the dedicated extra option for `ses` - see #4524 for why.